spotlight-api/core/metricsApi.js

```diff
@@ -242,9 +368,11 @@ function fetchSentencingMetrics(isDemo, tenantId, callback) {
 }
 
 module.exports = {
+  fetchMetricsByName,
   fetchParoleMetrics,
   fetchPrisonMetrics,
   fetchProbationMetrics,
   fetchSentencingMetrics,
   fetchRaceMetrics,
+  memoryCache,
```

Comment:
I don't love exporting this but I would feel better about it if we just put a lil' inline comment here that reads, "This is exported for the sake of unit testing and should never be used outside of this module!"

Reply:
oh right, I also don't love this. something I thought about (but I guess forgot to do?) would be to instead export a more limited "clear cache" function instead of the entire cache object, to limit the potential for future abuse. I think maybe I'll do that instead